### PR TITLE
Devtest v1

### PR DESCRIPTION
**Pull request by SaikumarRagala**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1 [hunk 1]
```diff
@@ -453,7 +451,7 @@ class SVTResourceResolver: AzSKRoot {
                         serviceConnections = -1;
                     };
 
-                    if($this.UsePartialCommits -and $this.DoNotRefetchResources){
+                    if($this.UsePartialCommits -and $this.DoNotRefetchResources -and $this.isDnrrAllProject -eq $false){
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1:454]: Why $this.isDnrrAllProject -eq $false in if flock if its getting changed in the next block only?

$this.isDnrrAllProject will always be false here

File: src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1 [hunk 2]
```diff
@@ -466,42 +464,34 @@ class SVTResourceResolver: AzSKRoot {
...
                     }
 
-                    if($this.IsPartialScanActive -and $this.nonScannedResources.Count -ne 0){
-                        #$this.PublishCustomMessage("Resuming scan from last commit")
-                        if ($this.BuildNames.Count -gt 0 -and ($this.ResourceTypeName -in ([ResourceTypeName]::Build, [ResourceTypeName]::All, [ResourceTypeName]::Build_Release, [ResourceTypeName]::Build_Release_SvcConn_AgentPool_VarGroup_User_CommonSVTResources))) {
-                            $this.createBuildSVTResource($projectName,$organizationId,$projectId);
-                        }
-                        if ($this.ServiceConnections.Count -gt 0 -and ($this.ResourceTypeName -in ([ResourceTypeName]::ServiceConnection, [ResourceTypeName]::All, [ResourceTypeName]::Build_Release_SvcConn_AgentPool_VarGroup_User_CommonSVTResources))){
-                            $this.createServiceConnectionSVTResource($projectName,$organizationId,$projectId,$projectData)
+                    if($this.IsPartialScanActive -and $this.nonScannedResources.Count -ne 0 -and $this.isDnrrAllProject -eq $false){
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1:467]: same as above

File: src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1 (hunk 1 above)
Comment (juhi037) [reply src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1:454]: Added this check to stop printing "Resuming scan from last commit. Fetching unscanned resources..." for the rest of the projects. Also when the loop starts for next project we do not need to get non scanned resources again. The tracker file is parsed in one go.

File: src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1 (hunk 2 above)
Comment (juhi037) [reply src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1:467]: When the loop starts for the next project, we need not load the releases again for any of the projects, all releases have been added to svt in first pass itself

**Review by Garima-msft — APPROVED** [review]

**Pull request merged**